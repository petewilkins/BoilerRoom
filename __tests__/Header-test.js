import 'react-native';
import React from 'react';
import Header from '../src/components/Header';

import renderer from 'react-test-renderer';

test('renders the Header component correctly', () => {
  const tree = renderer.create(
    <Header />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
