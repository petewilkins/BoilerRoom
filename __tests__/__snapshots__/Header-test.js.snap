// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Header component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F8F8F8",
      "elevation": 2,
      "height": 60,
      "justifyContent": "center",
      "paddingTop": 15,
      "position": "relative",
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.2,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
      }
    }
  />
</View>
`;
